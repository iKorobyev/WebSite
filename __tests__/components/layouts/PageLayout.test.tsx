import React from 'react'
import { cleanup, render, waitFor, act } from '@testing-library/react'
import userEvent from '@testing-library/user-event'
import { PageLayout } from '~layouts'

// beforeAll(() => {})
// afterAll(() => {})
// beforeEach(() => {})
afterEach(() => {
  cleanup()
  jest.clearAllMocks()
})

describe('PageLayout', () => {
  it('renders correctly', async () => {
    const props = {}

    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
    /* @ts-ignore */
    const { container } = render(<PageLayout {...props} />)
    await waitFor(() => container)

    expect(container).toMatchSnapshot()
  })
})
