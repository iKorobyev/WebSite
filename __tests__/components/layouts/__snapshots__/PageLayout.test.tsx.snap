// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageLayout renders correctly 1`] = `
<div>
  <main
    class="flex-1"
  >
    <div
      class="container"
    />
  </main>
</div>
`;
